src/states/BattleResolveState.py

```diff
@@ -90,134 +90,141 @@ def update(self, dt, events):
+                        'effectOrder': self.effectOrder,
+                        'effect': effect,
+                        'effectOwner': effectOwner
+                    })
+                case EffectType.CLEANSE:
+                    pass
+                case EffectType.DISCARD:
+                    pass
+                case EffectType.SAND_THROW:
+                    pass
+                case EffectType.ANGEL_BLESSING:
+                    pass
+                case EffectType.DESTINY_DRAW:
+                    pass
+                case EffectType.RESET_HAND:
+                    pass
+                # WARRIOR CLASS
+                case EffectType.WARRIOR:
+                    if len(self.player.buffs) >= 1 and self.player.job == PlayerClass.WARRIOR:
+                        g_state_manager.Change(SelectionState.MOVE, {
+                        'player': self.player,
+                        'enemy': self.enemy,
+                        'field': self.field,
+                        'turn': self.turn,
+                        'currentTurnOwner': self.currentTurnOwner,
+                        'effectOrder': self.effectOrder,
+                        'effect': effect,
+                        'effectOwner': effectOwner
+                    })
+                case EffectType.BLOOD_SACRIFICE:
+                    hp_paid = math.floor(self.player.health * 0.3)
+                    self.player.health -= hp_paid
+                    buffList = self.getBuffListFromEffect(effect)
+                    buffList[0].value[0] = hp_paid
+                    self.player.add_buffs(buffList)
+                # RANGER CLASS
+                case EffectType.CRITICAL:
+                    chance = random.randint(1, 6)
+                    threshold = 2
+                    for buff in self.player.buffs:
+                        if buff.name == 'Critical+ (4/6 chance)':
+                            threshold = 4
+                            break
+                    print(f'{chance} <= {threshold}')
+                    if chance <= threshold:
+                        print('Critical hit!')
+                        critical_buff = Buff(CARD_BUFF['critical_buff'])
+                        critical_buff.value[0] = self.player.selectedCard.attack // 2
+                        self.player.add_buffs([critical_buff])
+                # MAGE CLASS
+                case EffectType.TRUE_DAMAGE:
+                    pass
+                case EffectType.NEXT_MULTI:
+                    pass
+                # BOSSES
+                case EffectType.KAMIKAZE:
+                    pass
+                case EffectType.SPAWN:
+                    pass
+                case EffectType.HEAL:
+                    pass
+                case EffectType.COPY:
+                    pass
+                case _:
+                    print(f'Unknown effect type: {effect.type}')
+        else:
+            print('stunt')
```

Comment:
is there any visual feedback like rendering stunt message on the game? or is it just printing on the terminal?

Reply:
at the moment, it is just printing on the terminal.

Reply:
ok, might leave it as future improvement kub